### PR TITLE
Doc quailty

### PR DESCRIPTION
**Pull request by allclark**

### CODE REVIEW

File: src/command_modules/azure-cli-context/azure/cli/command_modules/context/_help.py [hunk 1]
```diff
@@ -9,35 +9,35 @@
 
 helps['context'] = """
     type: group
-    short-summary: Manage contexts
+    short-summary: Manage storage contexts
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-context/azure/cli/command_modules/context/_help.py]: These commands are not related to storage.

File: src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_help.py [hunk 2]
```diff
@@ -34,4 +34,13 @@
...
 
+helps['account set'] = """
+    type: command
+    short-summary: Sets the active (default) subscription
+"""
 
+helps['account show'] = """
+    type: command
+    short-summary: Show the details of a subscription
+    long-summary: If the subscription isn't specified, shows the details of the default suscription
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_help.py]: typo here: subscription

**Comment by allclark** [comment]
@derekbekoe Thanks for catching that.
